docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -428,18 +440,17 @@ SELECT * FROM logs WHERE has(mapValues(attributes), '192.168.1.1'); -- fast
 ### Direct read {#direct-read}
 
 Certain types of text queries can be speed up significantly by an optimization called "direct read".
-More specifically, the optimization can be applied if the SELECT query does _not_ project from the text column.
```

Comment:
Hmm, is it not a prerequisite for direct read that the query doesn't project from the indexed column?